### ISSUE TITLE
ConnectionRefusedError: [Errno 61] Connection refused

### ISSUE THREAD

**Issue by Tang-SL** [ejeschke/ginga#992]
Hi there, my issue is similar to #958, but simply getting this [Errno 61] when trying to do `ggrc help`
I can open a ginga window with `ginga`, but no response from `ginga --modules=RC`
I am using MacOS BigSur v11.6, and python 3.8, detailed error message is attached. thank you in advance!
<img width="858" alt="Screen Shot 2022-02-15 at 17 11 04" src="https://user-images.githubusercontent.com/49220392/154019806-246182a7-1384-4ded-94da-2ba135cbc22f.png">
!

**Comment by ejeschke** [comment]
@Tang-SL, following the advice in #958, what do you get when you do the following in Python:
```python
import socket
print(socket.gethostname())
```
?

**Comment by Tang-SL** [comment]
@ejeschke Thanks for the very prompt reply! it returns me with: cosdhcp012.icrr.u-tokyo.ac.jp
And I have checked my /etc/hosts , which shows:
<img width="413" alt="Screen Shot 2022-02-16 at 13 12 06" src="https://user-images.githubusercontent.com/49220392/154194756-200352bd-92a3-433e-9301-15a0ddb2da52.png">

**Comment by ejeschke** [comment]
@Tang-SL, what happens if you invoke `ggrc` like this:
```bash
$ ggrc --host=127.0.0.1
```
?

**Comment by Tang-SL** [comment]
Still shows the same error 61.

**Comment by ejeschke** [comment]
Ok, can you see the "RC" tab in the Ginga UI?

**Comment by ejeschke** [comment]
Based on the other issue you referenced, we probably have one of the following issues:
- firewall is enabled on the mac, so that we cannot access port 9000
- port 9000 is already used by another application
- localhost is not defined (although your /etc/hosts indicates that it should be)

If you can access the RC tab, change the port number the "Set Addr" field and press Set, then Restart.  Then in ggrc, use --port=NNNN to use the same port number.

If this works, it can be set in your $HOME/.ginga startup folder using the procedure described in the other issue.

**Comment by Tang-SL** [comment]
Excuse me, I found my issue is somewhat related to the environment, in one environment, `ginga` works, `ginga --modules=RC` also works, RC tab shows Addr: localhost:9000, but `ggrc help` sometimes work, sometimes do not...![Screen Shot 2022-02-17 at 13 29 53](https://user-images.githubusercontent.com/49220392/154405602-0bf72eb6-2c95-4329-a8eb-837f12d965da.png)

In another environment, `ggrc help` is in the same situation, but both `ginga` and `ginga --modules=RC` do not show anything.

**Comment by Tang-SL** [comment]
After I added plugin_RC.cfg to my ~/.ginga folder following #958 , now `ggrc help` works when ginga window is opening. 

However, I found my issue seems to be somewhat related to the environment. When I shift to another environment, in which I really wanted ginga to work, both `ginga --modules=RC` and `ginga` show nothing, no error message, and no windows pops up.

**Comment by ejeschke** [comment]
Can you add the following to the command line when starting ginga?
```bash
--loglevel=20 --stderr --log=error.log
```
What kind of errors are reported in the log?

BTW, [this discussion](https://github.com/ejeschke/ginga/discussions/989) has a good description of how to create an environment for running ginga.  You might check that discussion to see if you are missing some package.

**Comment by Tang-SL** [comment]
No error is reported but just stuck at some stage.
[error.log](https://github.com/ejeschke/ginga/files/8086346/error.log)

**Comment by Tang-SL** [comment]
And also no conflicts reported from the packages installation.
<img width="1068" alt="Screen Shot 2022-02-17 at 16 19 27" src="https://user-images.githubusercontent.com/49220392/154425149-7e155306-70a7-4a9c-a9dd-f8b653440bcb.png">

**Comment by ejeschke** [comment]
Nothing unusual in the log, but it should end with "entering mainloop...".  How did you install ginga; by "pip install ginga" or from downloading source?  A new version was released today (3.3.0), if you installed from pip, it might be worth trying to upgrade by "pip install --upgrade ginga".

Can you find any notable differences between the two environments?

For what it is worth, I just tested on mac os x "monterey" 12.2 with --modules=RC and ggrc and it is working fine there. Previously I think I was running "big sur" that was also fine.

**Comment by Tang-SL** [comment]
I fixed it out to be the `PyQt5-sip` package, major update from 4.19.18 to 12.9.1 gets it to work, problem solved! Thanks a lot for your help!
